src/main/java/domain/Money.java

```diff
@@ -1,25 +1,27 @@
+package domain;
+
 public class Money {
-    private int value;
+    private final int value;
 
-    public static Money of(String input) {
+    public static Money of(final String number) {
```

Comment:
```suggestion
    public static Money from(final String number) {
```
정적 팩터리 메서드를 사용할 때의 네이밍 컨벤션을 알아보아요!